src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -97,8 +84,6 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
 
     draft_msg = '[UNRELEASED DRAFT]'
     msg_chunks = ()
```

Comment:
Before this, there was a chance of having scm part combined with draft by concatenation. Now that there's always just one option (draft), there's no need for the tuple, nor for the if-block at all.